packages/editor/src/client/base.tsx

```diff
@@ -75,14 +75,31 @@ export function Base({children}: BaseProps) {
       <Container>
         <Sidebar
           style={{display: 'flex', flexDirection: 'column'}}
+          appearance="default"
           width={isExpanded ? 260 : 56}
           collapsible>
           <Sidenav
             expanded={isExpanded}
             defaultOpenKeys={['1']}
-            appearance="subtle"
+            appearance="default"
             style={{flex: '1 1 auto'}}>
             <Sidenav.Body>
+              <IconButton
+                style={{
+                  position: 'absolute',
+                  top: '5vh',
+                  left: isExpanded ? 260 : 56,
+                  transform: 'translate(-50%)',
+                  zIndex: 100
+                }}
+                className="collapse-nav-btn"
+                appearance="primary"
```

Comment:
Personal preference: I think it's okay with a subtle style now that it's higher up and more visible